conanfile_rtc.py

```diff
@@ -16,7 +16,7 @@ def package(self):
         relative = "3rdparty/PROJ/"
 
         # headers
-        self.copy("proj.h", src=base + "src", dst=relative)
+        self.copy("src/proj.h", src=base + "src", dst=relative)
```

Comment:
The base variable already adds `src` so adding it again would give you `src=${path} + "src".  Can you test this locally first as I think the header is not going to go where you expect it to.